submissions/nataliaArtsiukh/DocumentObjectModel/script.js

```diff
@@ -0,0 +1,72 @@
+window.onload = function () {
```

Comment:
Leave only the function for the initial display of information. Everything else can be moved outside the function.